index.html

```diff
@@ -10,131 +10,103 @@
+        <img  class="prestigeunlock img offering" src="Pictures/Offering.png" title='Offering'  alt="">
+        <div class="prestigeunlock statDisplay" id="offeringDisplay" >1</div>
+        <img class="transcendunlock img quark" src="Pictures/Quark.png" title='Quark' alt="">
+        <div class="transcendunlock statDisplay" id="quarkDisplay">1</div>
+        <img class="transcendunlock img mythos" src="Pictures/Mythos.png" title='Mythos' alt="">
+        <div class="transcendunlock statDisplay" id="mythosDisplay">1</div>
+        <img class="transcendunlock img mythosshard" src="Pictures/MythosShard.png" title='Mythos Shard' alt="">
+        <div class="transcendunlock statDisplay" id="mythosshardDisplay">1</div>
+        <img class="reincarnationunlock img particle" src="Pictures/Particle.png" title='Particle'  alt="">
+        <div class="reincarnationunlock statDisplay" id="particlesDisplay">1</div>
+        <img class="reincarnationunlock img obtanium" src="Pictures/Obtainium.png" title='Obtainium' alt="">
+        <div class="reincarnationunlock statDisplay" id="obtainiumDisplay">1</div>
+        <div id="resetrow">
+        
+            <img class="coinunlock4" id="prestigebtn" src="Pictures/Transparent Pics/Prestige.png" alt="">
+            <img class="prestigeunlock" id="transcendbtn" src="Pictures/Transparent Pics/Transcend.png" alt="">
+            <img class="transcendunlock" id="reincarnatebtn" src="Pictures/Transparent Pics/Reincarnate.png" alt="">
+            <img class="transcendunlock" id="acceleratorboostbtn" src="Pictures/Transparent Pics/AcceleratorBoost.png" alt="">
+            <img class="transcendunlock" id="challengebtn" src="Pictures/Transparent Pics/Challenge.png" alt="">
+            <img class="reincarnationunlock" id="reincarnatechallengebtn" src="Pictures/Transparent Pics/ChallengeReincarnation.png" alt="">
+            <img class="ascendunlock" id="ascendChallengeBtn" src="Pictures/Transparent Pics/ChallengeAscension.png" alt="">
+            <img class="chal10" id="ascendbtn" src="Pictures/questionable.png" alt="">
+            
+            <div id="resetrewards" style="display: block;">
+                <div>
+                    <img id="resetofferings1" src="" style="font-size: 0;" alt="">
+                    <div id="resetofferings2" style="color: gold"></div>
+                    <img id="resetcurrency1" src="" style="font-size: 0;" alt="">
+                    <div id="resetcurrency2" style="color:gold"></div>
+                    <img id="resetobtainium" src="" alt="">
+                    <div id="resetobtainium2" style="color:pink;"></div>
+                </div>
+            </div>
+        </div>
+        <div id="resetinfo" style="color:white"></div>
+        <nav id="navbar">
+        
+            
+            <ul id="tabrow">
+                <li id="buildingstab">Buildings</li>
+                <li id="upgradestab">Upgrades</li>
+                <li id="settingstab">Settings</li>
```

Comment:
The _only_ issue I've noticed is that this tab is incorrectly placed (likely due to absolute positioning in the original html?). Once this gets moved between the corruption/shop tabs this is good to merge.